examples/b2c/checkout/integrations/classes/B2BCheckInventorySampleTest.cls

```diff
@@ -3,7 +3,7 @@ public class B2BCheckInventorySampleTest {
     @testSetup static void setup() {
         Account account = new Account(Name='TestAccount');
         insert account;
-        WebStore webStore = new WebStore(Name='TestWebStore', DefaultLanguage='en_US');
+        WebStore webStore = new WebStore(Name='TestWebStore', SupportedLanguages='en_US', DefaultLanguage='en_US');
```

Comment:
When only support en_US? i thought you also support jp?

Reply:
This is just a test. And no language functionality is being tested. "Supported Languages" was added because these tests started failing after we made the translation workbench changes.